### DISCUSSION TITLE
H100 PCIe is significantly worse than the H100 SXM

### DISCUSSION THREAD

**Discussion by bobqianic** [ggml-org/llama.cpp#8511] [General]
Apparently, the H100 PCIe is significantly worse than the H100 SXM.

On the H100 SXM, we can easily achieve `620-730 TFLOPs` (BF16/FP16). In contrast, on the H100 PCIe, we can only reach `350-450 TFLOPs` (BF16/FP16).

The cheapest H100 SXM instance I found is on Lambda Cloud, where 8xH100 SXM GPUs cost `$27.92` per hour (before tax), roughly `$3.49` per GPU hour. The H100 PCIe offers only `56.4%-61.6%` of the H100 SXM's performance, so the fair charge for H100 PCIe should be around `$1.96-$2.15` per GPU hour. However, the price of H100 PCIe on Lambda Cloud is `$2.49` per GPU hour, which is `15%-27%` more expensive than it should be.

Therefore, my advice is if you have serious training workloads, you should use the H100 SXM instead of the H100 PCIe. The name H100 PCIe is misleading. It should be called **H50** or **H60**, as its performance is only `56.4%-61.6%` of the H100 SXM : (

<img width="481" alt="6db2ed0871a645f9b23e6f85fc0b81b" src="https://github.com/user-attachments/assets/de124c50-1f77-4dbc-a342-9d66d8226bed">
<img width="484" alt="6bb53af5370fec78b6a035c6518baf6" src="https://github.com/user-attachments/assets/f44f859f-78e8-4810-9de0-0764b9391d1b">

> 

<img width="475" alt="9ecb9969968bb5623d0d359f26ae192" src="https://github.com/user-attachments/assets/985a112b-ae72-4ab0-b13d-0430326bf8df">
<img width="432" alt="a5244b4f44f1c8d390debfaed75ba71" src="https://github.com/user-attachments/assets/aed4f9fc-a201-4c61-896e-1051bcb489d8">

**Comment by Leikoe** [comment]
Also remember that sxm versions are 700W and are SIGNIFICANTLY harder to cool. Some people showed that on extended training runs performance degrades over time because the rig thermal throttles.
